data.sql

```diff
@@ -1,5 +1,34 @@
 /* Populate database with sample data. */
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
- INSERT INTO animals VALUES (2, 'Gabumon', '2028-11-15', 2, true,8);
-INSERT INTO animals VALUES (3, 'Pikachu', '2021-01-07',1,false,15.04);
- INSERT INTO animals VALUES (4, 'Devimon', '2017-03-12',5,false,11);
\ No newline at end of file
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Agumon', '2020-02-03', 0, true, 10.23);
+
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Gabumon', '2028-11-15', 2, true,8);
+
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Pikachu', '2021-01-07',1,false,15.04);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Devimon', '2017-03-12',5,false,11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species) 
+VALUES ('Charmander','2020-02-08',0,false,-11,'unknown');
+
+INSERT INTO animals 
+VALUES ('Plantmon', '2021-11-15', 2, true, -5.7, 'unknown');
+
+INSERT INTO animals 
+VALUES ('Squirtle', '1993-04-02', 3, false, -12.13, 'unknown');
+
+INSERT INTO animals 
+VALUES ('Angemon', '2005-06-12', 1, true, -45, 'unknown');
+
+INSERT INTO animals 
+VALUES ('Boarmon', '2005-06-07', 7, true, 20.4, 'unknown');
+
+INSERT INTO animals 
+VALUES ('Blossom', '1998-10-13', 3, true, 17, 'unknown');
+
+INSERT INTO animals 
+VALUES ('Ditto', '2022-05-14', 4, true, 22, 'unknown');
```

Comment:
- [ ] Great job removing the id columns, however you also need to remove the species column. The only asked for columns are `name`, `date_of_birth`, `escape_attempts`, `neutered`, and `weight_kg` columns see a correct version of your code below:

    ```
    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Charmander', '2020/2/8', '0', 'false', '-11.0');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Plantmon', '2021/11/15', '2', 'true', '-5.7');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Squirtle', '1993/04/02', '3', 'false', '-12.13');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Angemon', '2005/06/12', '1', 'true', '-45.0');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Boarmon', '2005/06/07', '7', 'true', '20.4');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Blossom', '1998/10/13', '3', 'true', '17.0');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Ditto', '2022/05/14', '4', 'true', '22.0');

```